crates/store/re_data_source/src/data_source.rs

```diff
@@ -97,35 +98,39 @@ impl DataSource {
             //
             // In order to avoid having to swallow errors based on unreliable heuristics (or inversely:
             // throwing errors when we shouldn't), we just make reading from standard input explicit.
-            if uri == "-" {
-                return Self::Stdin;
+            if url == "-" {
+                return Some(Self::Stdin);
             }
 
-            let path = std::path::Path::new(&uri).to_path_buf();
+            let path = std::path::Path::new(url).to_path_buf();
 
-            if uri.starts_with("file://") || path.exists() {
-                return Self::FilePath(_file_source, path);
+            if url.starts_with("file://") || path.exists() {
+                return Some(Self::FilePath(_file_source, path));
             }
 
-            if looks_like_a_file_path(&uri) {
-                return Self::FilePath(_file_source, path);
+            if looks_like_a_file_path(url) {
+                return Some(Self::FilePath(_file_source, path));
             }
         }
 
-        if let Ok(uri) = uri.as_str().parse::<RedapUri>() {
-            return Self::RerunGrpcStream {
+        if let Ok(uri) = url.parse::<RedapUri>() {
+            Some(Self::RerunGrpcStream {
                 uri,
                 select_when_loaded: true,
-            };
+            })
+        } else if url.ends_with(".rrd") || url.ends_with(".rbl") {
+            Some(Self::RrdHttpUrl {
+                url: url.to_owned(),
+                follow: false,
+            })
+        } else {
+            None
```

Comment:
This seems like a welcome change, but I sure do hope nobody depends on our previous assumption here